README.md

```diff
@@ -22,240 +22,286 @@ $ go get -u github.com/census-instrumentation/opencensus-go
+    // handle error
+}
+if err := stats.UnregisterView(myView2); err != nil {
+    // handle error
+}
+```
 
 ### To subscribe/unsubscribe to a view's collected data
 Once a subscriber subscribes to a view, its collected date is reported at a regular interval. This interval is configured system wide.
 
 Subscribe to a view:
 
-    c1 := make(c chan *ViewData)
-    if err := stats.SubscribeToView(myView1, c1); err != nil {
-        // handle error
-    }
-
-    c2 := make(c chan *ViewData)
-    if err := stats.SubscribeToView(myView2, c2); err != nil {
-        // handle error
-    }
-    ... 
+```go
+c1 := make(c chan *stats.ViewData)
+if err := stats.SubscribeToView(myView1, c1); err != nil {
+    // handle error
+}
+c2 := make(c chan *stats.ViewData)
+if err := stats.SubscribeToView(myView2, c2); err != nil {
+    // handle error
+}
+```
 
 Unsubscribe from a view:
 
-    if err := stats.UnsubscribeFromView(myView1, c1); err != nil {
-        // handle error
-    }
-    if err := stats.UnsubscribeFromView(myView2, c2); err != nil {
-        // handle error
-    }    
-    ...
+```go
+if err := stats.UnsubscribeFromView(myView1, c1); err != nil {
+    // handle error
+}
+if err := stats.UnsubscribeFromView(myView2, c2); err != nil {
+    // handle error
+}    
+```
 
 Configure/modify the default interval between reports of collected data. This is a system wide interval and impacts all views. The default interval duration is 10 seconds. Trying to set an interval with a duration less than a certain minimum (maybe 1s) should have no effect.
 
-    d := 20 * time.Second
-    stats.SetReportingPeriod(d)
+```go
+d := 20 * time.Second
+stats.SetReportingPeriod(d)
+```
 
 ### To collect data for on-demand retrieveal
 Even if a view is registered, if it has no subscriber no data for it is collected. In order to retrieve data on-demand for view, either the view needs to have at least 1 subscriber or the libray needs to be instructed explicitly to collect collect data for the desired view.
 
-    // explicitly instruct the library to collect the view data for an on-demand retrieval.
-    if err := stats.ForceCollection(myView1); err != nil {
-        // handle error
-    }
-    ...
-
-    // Retrieve data. The returned rows are a []stats.Row. Each row is a struct:
-    //  {
-	//      Tags             []tags.Tag
-	//      AggregationValue AggregationValue
-    //  }
-    rows, err := stats.RetrieveData(v View)
-    if err != nil {
-        // handle error
-    }
-    ...
-
-    // explicitly instruct the library to stop collecting the view data for the
-    // on-demand retrieval. Has no impact on subscriptions, and if the view
-    // still has subscribers, the data for the view will still keep being
-    // collected.
-    if err := stats.StopForcedCollection(myView1); err != nil {
-        // handle error
-    }
-    ...
-
-## Recording usage/measurements
+```go
+// explicitly instruct the library to collect the view data for an on-demand retrieval.
```

Comment:
Keep the Go comments as proper sentences with regular punctuation and capitalization rules.

// Explicitly instruct the library to collect the view data for an on-demand retrieval.

Reply:
Done